test/notifications/receive_test.dart

```diff
@@ -34,7 +34,7 @@ void main() {
   group('tokens', () {
     test('APNs token registration using correct app bundle ID', () async {
       await init();
-      check(await testBinding.getAppBundleId())
+      check((await testBinding.packageInfo)?.packageName?? 'com.zulip.flutter.test')
```

Comment:
Falling back to `com.zulip.flutter.test` defeats the purpose of this test. I think we don't expect `packageInfo` to be missing in this case, right?

Reply:
You are correct, falling back to the literal defeats the purpose. Will update.